stable/hive/templates/v3io-config-configmap.yaml

```diff
@@ -0,0 +1,153 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-v3io-config
+  labels:
+    app: {{ template "hive.name" . }}
+    chart: {{ template "hive.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+data:
+{{ include "v3io-configs.java.configMap" . | indent 2 }}
+
+  health_check.sh: |
+{{ include "v3io-configs.script.javaHealthCheck" . | indent 4 }}
+    # Check if service is listening on the specified protocol and port
+    PROTOCOL="tcp"
+    HOST="127.0.0.1"
+    PORT={{ .Values.metastore.containerPort }}
+
+    for i in "$@"
+    do
+    case $i in
+        -t=*|--type=*)
+        PROTOCOL="${i#*=}"
+        ;;
+        -h=*|--host=*)
+        HOST="${i#*=}"
+        ;;
+        -p=*|--port=*)
+        PORT="${i#*=}"
+        ;;
+    esac
+    done
+
+    RC=echo < /dev/$PROTOCOL/$HOST/$PORT
+    if [ "$RC" -ne "0" ]; then
+      echo 'Unable to communicate with "$HOST:$PORT" over "$PROTOCOL".'
+      exit $RC
+    fi
+
+  v3io-hive.sh: |
+    #!/usr/bin/env bash
+    set -e
+
+    echo alias ll=\"ls -lah --color\" >> ~/.bash_aliases
+
+{{ include "v3io-configs.script.lookupService" . | indent 4 }}
+
+    V3IO_HIVE_CONFIG={{ .Values.configPath }}
+    mkdir -p $V3IO_HIVE_CONFIG
+
+    cp -r $HIVE_HOME/conf/* $V3IO_HIVE_CONFIG/
+    cp {{ .Values.global.v3io.configMountPath }}/hive-site.xml $V3IO_HIVE_CONFIG/
```

Comment:
I don't understand these steps. You can use `$HIVE_CONFIG` without the V3IO flavour